.changeset/brave-gorillas-begin.md

```diff
@@ -0,0 +1,6 @@
+---
+'@twilio-paste/user-dialog': patch
+'@twilio-paste/core': patch
+---
+
+Removal of ts annotation to expect an error for reset button style overrides
```

Comment:
```suggestion
[User Dialog] Removal of ts annotation to expect an error for reset button style overrides
```

Reply:
Although I wonder if we even need to ship this as a patch since it doesn't affect users really?

Reply:
Probably not, it's just to appease the Danger JS check